bundle/Resources/views/app/item/nglayouts_app_preview.html.twig

```diff
@@ -0,0 +1,23 @@
+{% if content.hasField('image') and not content.fields.image.empty %}
+    <div class="image">
+        {% if content.fields.image.fieldTypeIdentifier is same as 'ngremotemedia' %}
+            {% set variation = netgen_remote_variation(content.innerContent, 'image', 'nglayouts_app_preview') %}
+
+            <img src="{{ variation.url }}" />
+        {% elseif content.fields.image.fieldTypeIdentifier is same as 'ezimage' %}
+            {% set image_alias = ng_image_alias(content.fields.image, 'nglayouts_app_preview') %}
```

Comment:
`ng_image_alias` is available only in Site API, so should not be used here.

Reply:
Changed in https://github.com/netgen-layouts/layouts-remote-media/pull/13